packages/cli/src/actions/start.ts

```diff
@@ -15,10 +15,12 @@ const start: StartHandler = async (filename: string, opts: any) => {
     spinner.fail('Please specify the config path');
     return process.exit(1);
   }
-  filename = path.isAbsolute(filename) ? filename : path.join(process.cwd(), filename);
-  if (!existsSync(filename)) {
-    spinner.fail(`${filename} not exists`);
-    return process.exit(1);
+  if (!isUrl(filename)) {
```

Comment:
Why do we add this check?

Reply:
If not, the file url will be joined the file path prefix, we should skip this step if it's a url.

Reply:
How about moving the existence check to daemon? So here is to do convert filename to file-protocol URI, and the daemon API might handle the URI only.

Reply:
> How about moving the existence check to daemon? So here is to do convert filename to file-protocol URI, and the daemon API might handle the URI only.

It looks more reasonable.